projectile.el

```diff
@@ -1234,28 +1234,37 @@ function is executing."
   "First remove ignored files from FILES, then add back unignored files."
   (projectile-add-unignored (projectile-remove-ignored files)))
 
-(defun projectile-remove-ignored (files &optional subdirectories)
+(defun stringi= (string1 string2)
+  "compare two strings case insensitively"
+  (equal (compare-strings string1 nil nil string2 nil nil t) t))
+
+(defun projectile-remove-ignored (files)
   "Remove ignored files and folders from FILES.
 
-Operates on filenames relative to the project root.  Optionally,
-you can filter ignored files in subdirectories by setting
-SUBDIRECTORIES to a non-nil value."
-  (let ((ignored (append (projectile-ignored-files-rel)
-                         (projectile-ignored-directories-rel))))
+Operates on filenames relative to the project root."
```

Comment:
The changes you've made in this file could benefit from a few explanatory comments.